workflow/automation/execution_scripts/add_to_mgmt_queue.py

```diff
@@ -44,4 +81,24 @@
         const.Status.from_str(args.status).value,
         job_id=args.job_id,
         error=args.error,
+        start_time=int(
+            datetime.strptime(args.start_time, "%Y-%m-%d_%H:%M:%S").timestamp()
+        )
+        if args.start_time is not None
+        else None,
```

Comment:
These 3 "a if b else c" statements need temporary variables, this is way too hard to read